test/Microsoft.AspNetCore.SignalR.Client.Tests/ConnectionTests.cs

```diff
@@ -479,7 +480,7 @@ public void ConnectionReturnsUrlUsedToStartTheConnection()
                 .Returns<HttpRequestMessage, CancellationToken>(async (request, cancellationToken) =>
                 {
                     await Task.Yield();
-                    if (request.RequestUri.AbsolutePath.EndsWith("/send"))
+                    if (request.Method == HttpMethod.Post)
```

Comment:
You can use the HttpMethods class here. 
https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNetCore.Http.Abstractions/HttpMethods.cs#L50